### PR TITLE
pret is not capitalized

### PR DESCRIPTION
**Pull request by vulcandth**
Precedence is that pret is not capitalized.

Edit: Well.. most of the time that is.

### CODE REVIEW

**Comment by Rangi42** [comment]
While you're modifying this file, can you make some other changes?

```diff
-Some would say the name stands for "Pokémon Reverse Engineering Team". We consider those people "misguided".
```

```diff
-pret is not a community.
```

```diff
-If we had a vanity, it would be [discord.gg/pret](https://discord.gg/d5dubZ3).
+Discord: [pret](https://discord.gg/d5dubZ3).
+IRC: [libera#pret](https://web.libera.chat/?#pret)
```

Basically make it more like the info all the repos have/had, just straightforward.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
 
-## Discord invite
-
-If we had a vanity, it would be [discord.gg/pret](https://discord.gg/d5dubZ3).
-
-## Community pages
-
-PRET is not a community.
+* Discord: [pret](https://discord.gg/d5dubZ3).
+* IRC: [libera#pret](https://web.libera.chat/?#pret)).
```
Comment (dannye) [inline README.md]: Extra closing parenthesis here (also, do these lines need to end with a period?)

**Comment by dannye** [comment]
Just a thought, wondering how others feel..

The gb(c) repos are currently listed chronologically, like:
* Pokémon Red and Blue
* Pokémon Gold and Silver (Space World '97 demo)
* Pokémon Yellow
* Pokémon Trading Card Game
* Pokémon Pinball
* Pokémon Gold and Silver
* Pokémon Crystal
* Pokémon Trading Card Game 2

What about a logical grouping, like:
* Pokémon Red and Blue
* Pokémon Yellow
* Pokémon Gold and Silver (Space World '97 demo)
* Pokémon Gold and Silver
* Pokémon Crystal
* Pokémon Trading Card Game
* Pokémon Trading Card Game 2
* Pokémon Pinball

**Comment by vulcandth** [comment]
What about including the associated discord channels as well; like:

* Pokémon Red and Blue -  [Discord #pokered]
* Pokémon Yellow - [Discord #pokered]
* Pokémon Gold and Silver (Space World '97 demo) - [Discord #prototypes-and-leaks]
* Pokémon Gold and Silver - [Discord #pokecrystal]
* Pokémon Crystal - [Discord #pokecrystal]
...

**Comment by dannye** [comment]
Yeah not a bad idea. I'm not too strongly for or against it.

**Comment by AnonymousRandomPerson** [comment]
@vulcandth Are you planning to add the Discord channels to this PR per your comment? Or will that be a separate PR?

**Comment by AnonymousRandomPerson** [comment]
Remaking this PR with the new comments addressed, as @vulcandth seems to be inactive here. https://github.com/pret/pret.github.io/pull/4

**Pull request closed without merging**